packages/mds-policy-author-api/tests/api.spec.ts

```diff
@@ -26,23 +26,23 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-enable prettier/prettier */
 /* eslint-enable @typescript-eslint/no-unused-vars */
-import { ApiServer } from '@mds-core/mds-api-server';
-import { GeographyFactory, GeographyServiceClient, GeographyServiceManager } from '@mds-core/mds-geography-service';
+import { ApiServer } from '@mds-core/mds-api-server'
+import { GeographyFactory, GeographyServiceClient, GeographyServiceManager } from '@mds-core/mds-geography-service'
 // eslint-disable-next-line prettier/prettier
 import type { PolicyDomainCreateModel, PolicyMetadataDomainModel } from '@mds-core/mds-policy-service';
```

Comment:
this is weird line. @avatarneil , that was causing all my `;` to be removed then immediately replaced.  
Removed it, it's fine now.